### PR TITLE
Add scenarios to beta documentation. Closes #430.

### LINKED ISSUE #430: Provide documentation for several basic scenarios for the Beta
## Specify type:
- Enhancement

## Description:
Provide basic example for login, creating a cred, source, and scan

### PR DESCRIPTION
**Pull request by chambridge**

### CODE REVIEW

**Comment by chambridge** [comment]
@jlprevatt This PR adds a basic set of scenarios for creating credentials, sources, running a scan and obtaining a report. Inspired by the Rho QSG. 🥇

File: docs/source/beta.rst [hunk 1]
```diff
@@ -208,3 +208,134 @@ Configuring the Command Line
...
+
+If you did not use the ``sshkeyfile`` option to provide an SSH key for the username value, enter the password of the user with root-level access at the connection password prompt.
+
+For example, for a Network credential where the name is **roothost1**, the user with root-level access is ``root``, and the SSH key for the user is in the ``~/.ssh/id_rsa`` file, you would enter the following command::
+
+  # qpc cred add --type network --name roothost1 --username root --sshkeyfile ~/.ssh/id_rsa
+
+qpc also supports privilege escalation with the ``become-method``, ``become-user``, and ``become-password`` options to create a Network credential for a user to obtain root-level access. You can use the ``become-*`` options with either the ``sshkeyfile`` or the ``password`` option.
+
+For example, for a Network credential where the is **sudouser1**, the user with root-level access is ``sysadmin``, and the access is obtained through the password option, you would enter the following command::
```
Comment (abaiken) [inline docs/source/beta.rst]: for a Network credential where the name is **sudouser1**

File: docs/source/beta.rst [hunk 2]
```diff
@@ -208,3 +208,134 @@ Configuring the Command Line
...
+
+After you enter this command, you are prompted to enter two passwords. First, you would enter the connection password for the username user, and then you would enter the password for the **become-method** which is the ``sudo`` command by default.
+
+2. Create at least one Network source that specifies one or more network identifiers, such as a host name, an IP address, a list of IP addresses, or an IP range, and one or more network credentials to be used for the scan::
+
+  # qpc source add --type network --name source_name --hosts host_name_or_file --cred cred_name
+
+For example, for a Network source where the name is **mynetwork**, the network to be scanned is the **192.0.2.0/24** subnet, and the Network credentials that are used to run the scan are **roothost1** and **roothost2**, you would enter the following command::
+
+  # qpc source add --type --name mynetwork --hosts 192.0.2.[1:254] --cred roothost1 roothost2
```
Comment (abaiken) [inline docs/source/beta.rst]: `# qpc source add --type network`

File: docs/source/beta.rst [hunk 3]
```diff
@@ -208,3 +208,134 @@ Configuring the Command Line
...
+  # qpc cred add --type vcenter --name vcenter_admin --username admin --password
+
+2. Create at least one vCenter source that specifies a network identifiers, such as a host name or an IP address of the vCenter server, and one vCenter credential to be used for the scan::
+
+  # qpc source add --type vcenter --name source_name --hosts host_name --cred cred_name
+
+For example, for a vCenter source where the name is **myvcenter**, the vCenter server to be scanned is the **192.0.2.10**, and the vCenter credential used to run the scan is **vcenter_admin**, you would enter the following command::
+
+  # qpc source add --type --name myvcenter --hosts 192.0.2.10 --cred vcenter_admin
+
```
Comment (abaiken) [inline docs/source/beta.rst]: `# qpc source add --type vcenter`

File: docs/source/beta.rst [hunk 4]
```diff
@@ -208,3 +208,134 @@ Configuring the Command Line
...
+  # qpc cred add --type satellite --name satellite_admin --username admin --password
+
+2. Create at least one Satellite source that specifies a network identifiers, such as a host name or an IP address of the Satellite server, and one Satellite credential to be used for the scan::
+
+  # qpc source add --type satellite --name source_name --hosts host_name --cred cred_name
+
+For example, for a Satellite source where the name is **mysatellite6**, the Satellite server to be scanned is the **192.0.2.15**, and the Satellite credential used to run the scan is **satellite_admin**, you would enter the following command::
+
+  # qpc source add --type --name mysatellite6 --hosts 192.0.2.15 --cred satellite_admin
```
Comment (abaiken) [inline docs/source/beta.rst]: `#qpc source add --type satellite`

File: docs/source/beta.rst [hunk 5]
```diff
@@ -208,3 +208,134 @@ Configuring the Command Line
...
+---------------
+Now that everything is installed and configured you can begin to utilize the capabilities of Quipucords to gather information on your IT infrastructure.
+
+Quipucords inspects various sources using credentials. You must first identify what sources will be inspected from the supported list:
+
+- Network
+- vCenter server
+- Satellite
+
+Network sources are composed of IP address, IP ranges, or hostnames. vCenter server and Satellite is created with the IP address or hostname of the server. Sources additionally are composed of credentials. A Network source can have a list of credentials as it is expected that many may be needed for a broad IP range, whereas vCenter server and Satellite use a single credential.
```
Comment (abaiken) [inline docs/source/beta.rst]: vCenter server and Satellite sources are created with either the IP address or hostname of the server.

**Comment by chambridge** [comment]
@abaiken I've fixed the horrible mistakes I made in the document. 😈 

Per your question about `--disable-optional-products` flag, the intent of the beta scenarios is to provide the most basic information to get up and running. More advanced usage is then located in the man page.

**Comment by abaiken** [comment]
🥇

**Pull request merged**